### PR TITLE
MSTR-114: Add markdown editor

### PR DESCRIPTION
**Pull request by masmerino13**

### CODE REVIEW

File: docs/monster-ui/markdownEditor().md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+title: markdownEditor()
+
+# monster.ui.markdownEditor()
+The `monster.ui.markdownEditor()` function convert a simple textarea into a markdown editor.
```
Comment (azefiel) [inline docs/monster-ui/markdownEditor().md]:
```suggestion
The `monster.ui.markdownEditor()` method converts a simple textarea into a markdown editor.
```

File: docs/monster-ui/markdownEditor().md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Syntax
+```javascript
+monster.ui.markdownEditor(target[, options]);
+```
+
+### Parameters
+
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`target` | A jQuery object that will be converted to markdow editor. | `jQuery` | | `true`
```
Comment (azefiel) [inline docs/monster-ui/markdownEditor().md]: You should avoid writing in the future tense when describing parameters in general (as it makes it unnecessarily verbose).
```suggestion
`target` | A jQuery object to be converted into a markdown editor. | `jQuery` | | `true`
```

File: docs/monster-ui/markdownEditor().md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+`options` | This will be a JavaScript Object, see the plugin [markdownEditor docs](https://simplemde.com/) for reference. | `Object`([#/options](#options)) | | `false`
```
Comment (azefiel) [inline docs/monster-ui/markdownEditor().md]:
```suggestion
`options` | List of options used to instantiate a markdown editor (available [options](https://github.com/sparksuite/simplemde-markdown-editor#configuration)). | `Object`([#/options](#options)) | | `false`
```

File: docs/monster-ui/markdownEditor().md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Description
+The `monster.ui.markdownEditor()` method convert a simple textarea element into a powerful markdown editor which is specified by the `target` parameter. The toolbar can be customized or hidden completely, please see [markdownEditor docs](https://simplemde.com/) for reference.
```
Comment (azefiel) [inline docs/monster-ui/markdownEditor().md]:
```suggestion
The `monster.ui.markdownEditor()` method converts a simple textarea element specified by the `target` parameter into a powerful markdown editor. The toolbar can be customized or hidden completely, please see [markdownEditor docs](https://simplemde.com/) for reference.
```

File: src/js/main.js [hunk 5]
```diff
@@ -65,7 +65,8 @@ require.config({
 		'toastr': 'js/vendor/toastr-1.3.0',
 		'touch-punch': 'js/vendor/jquery.ui.touch-punch.min',
 		'validate': 'js/vendor/jquery.validate.min',
-		'wysiwyg': 'js/vendor/bootstrap.wysiwyg.min'
+		'wysiwyg': 'js/vendor/bootstrap.wysiwyg.min',
+		'simplemde': 'js/vendor/simplemde.min'
```
Comment (azefiel) [inline src/js/main.js]: This list is sorted alphabetically to make it easier to search visually so please try to keep it that way.

**Comment by masmerino13** [comment]
Hi @azefiel ,

Any help is always welcome!

Regards.

**Pull request merged**